packages/deployments/contracts/src/cli/init/config/testnet/production.ts

```diff
@@ -94,6 +94,46 @@ export const TESTNET_PRODUCTION_INIT_CONFIG: InitConfig = {
         },
       },
     },
+    {
+      name: "nextAlUSD",
```

Comment:
should match the canonical symbol
```suggestion
      name: "alUSD",
```